### DISCUSSION TITLE
AniPlus (English subtitles)

### DISCUSSION THREAD

**Discussion by lokigryn** [iptv-org/community#729] [Q&A]
**Channel Name:** aniplus
**Channel ID:** AniplusAsia.sg
**Country:** Philippines
**Language:** japan
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19742
